crates/sui-adapter-transactional-tests/tests/transfer_object/shared.exp

```diff
@@ -17,9 +17,10 @@ Version: 2
 Contents: test::m::S {id: sui::object::UID {id: sui::object::ID {bytes: fake(108)}}}
 
 task 4 'transfer-object'. lines 27-27:
-Error: Attempt to transfer object 0x745ea5d547883aa753c1f9ad61290dafd7a462cc162ae5ddf2ede94917e3bf49 that does not have public transfer. Object transfer must be done instead using a distinct Move function call.
+Error: Transaction Effects Status: Invalid command argument at 0. Immutable and shared objects cannot be passed by-value.
+Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: InvalidObjectByValue }, source: None, command: Some(0) } }
 
 task 5 'view-object'. lines 29-29:
 Owner: Shared
-Version: 2
+Version: 3
```

Comment:
Interesting that the version number changed, is that expected?

Reply:
Yes, because this error used to be caught by the transaction input checker, but is now checked at runtime.